cmd/jaeger/internal/extension/remotesampling/factory.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package remotesampling
+
+import (
+	"context"
+	"time"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/confighttp"
+	"go.opentelemetry.io/collector/extension"
+
+	"github.com/jaegertracing/jaeger/ports"
+)
+
+// componentType is the name of this extension in configuration.
+var componentType = component.MustNewType("remote_sampling")
+
+const (
+	HTTPPort                            = 12345
```

Comment:
No, currently we dont have any sampling endpoint in `/ports`.

But in `V1 Collector` : `cmd/collector/app/flags/flags.go`:`L193` : we are using `ports.CollectorHTTP(14268)` as `default`. I think I should use the same.

Reply:
14268 is reasonable, since its other function in v1 is receiving spans in Jaeger format and we don't really want to support that anymore in v2, only OTLP. But did you check which protocol is used by OTEL SDKs? I remember that Java SDK was using gRPC for sampling, not the HTTP. If nothing uses HTTP then we'd only want to stand up a gRPC endpoint.

Reply:
when you do check OTEL SDKs please post links to the code.